app/views/settings/index.slim

```diff
@@ -1,8 +1,8 @@
-- title "settings"
+- title "Settings"
```

Comment:
This should probably be single quotes as well since you changed a bunch of `"` later in the file.